domain/storage/types.go

```diff
@@ -72,3 +72,16 @@ func DefaultStoragePools(registry storage.ProviderRegistry) ([]*storage.Config,
 	}
 	return result, nil
 }
+
+// ModelDetails describes details about a model.
+type ModelDetails struct {
+	ModelUUID      string `db:"uuid"`
+	ControllerUUID string `db:"controller_uuid"`
+}
```

Comment:
Remove db tags (copy and paste from above).

Reply:
ugh, cut and paste fail